### ISSUE TITLE
Reduce powerloss when neutral, powerloss when enemied

### ISSUE THREAD

**Issue by jflessenkemper** [magnusulf/Factions#62]
Hey Magnus,

I was speaking to Naantam about powerloss reductions when a faction is neutral
So reduce the powerloss or stop powerloss if they are killed while neutral

In this case, two factions would have to be both enemied to each other reduce eachother's power,
This feature could create dynamic war on servers without moderation

I just looked at the code, and it wouldn't be hard to reconfigure territory shield damage reduction code into powerloss reduction code when someone is in their own land
I just don't know too much about the member funcs and java to send a PR

**Comment by jflessenkemper** [comment]
## Affected Plugin




- **Affected Plugin Name:** Factions
- **Affected Plugin Version:** 3.2.3

## Describe your Environment

- **Server Version:** 1.13.2

**Comment by magnusulf** [comment]
Your original message is a bit unclear.

Do you want to reduce (or disable) power loss when the relation between the damager and the damagee is neutral?
Or do you want to reduce (or disable) power loss when someone dies in their own land?
Or both? Or something else?

**Comment by jflessenkemper** [comment]
Hey @magnusulf,

Sorry for being unclear, I meant maybe having a power modifier for different situations instead of losing power on any death no matter the circumstances.

For instance:

If you're neutral and someone else is your enemy, you don't lose any power because you haven't chosen to be their enemy.

If you're attacking your enemy, and they have enemied you, then you will both lose power on death.

If you're neutral, and another faction is neutral, then if you kill them or if they kill you, nobody will lose power because you're both neutral.

Basically, you can only lose power if you're both enemies to each other.

This would allow servers to facilitate player-controlled wars instead of having moderators look after it.

**Comment by jflessenkemper** [comment]
I understand you can't attack players who you have not enemied, but if a server has server-wide pvp then anyone can attack anyone.
